mcu.sch

```diff
@@ -1728,6 +1728,8 @@ $Comp
 L Connector:Conn_01x05_Male J5
 U 1 1 608CDC8A
 P 5150 7250
+AR Path="/608CDC8A" Ref="J5"  Part="1" 
+AR Path="/608F155A/608CDC8A" Ref="J5"  Part="1" 
```

Comment:
Not sure about the KiCad file format, so I don't really know what those changes are, ideas?